website/www/site/content/en/contribute/practises.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Beam community’s practises"
+layout: "arrow_template"
+---
+
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Beam community’s practises
+
+{{< figure src="/images/community/messages-icon.svg" >}}
+
+## Knows, upholds, and reinforces the Beam community’s practices
+
+- They have a proven commitment to the project
+- They share their intentions with the community
+- They accept and integrate community feedback in their plans, designs, code, etc.
+- They earnestly try to make Beam better with their contributions
+
+In particular, if a code contributor:
+
+- They earnestly try to make Beam better with their own code
+- They earnestly try to make Beam better with code review
+- They accept and integrate feedback on their code
+- They know, follow, and enforce Beam’s practices while reviewing/merging code - style, documentation, testing, backward compatibility, etc.
+
+{{< figure src="/images/community/beam-logo-icon.svg" >}}
+
+## Knows, upholds, and reinforces the Apache Software Foundation code of conduct
+
+In particular, we manifestly strive to:
+
+- Be open
+- Be empathetic
+- Be welcoming
+- Be friendly
+- Be patient
+- Be collaborative
+- Be inquisitive
+- Be careful in the words that they choose
+
+[To learn more see the ASF documentation.](https://httpd.apache.org/docs/)
+
+{{< figure src="/images/community/diamond-icon.svg" >}}
+
+## Knows, upholds, and reinforces the responsibilities of an Apache Software Foundation committer
+
+- They help create a product that will outlive the interest of any particular volunteer (including themselves)
+- They grow and maintain the health of the Apache community
+- They help out with surrounding work, such as the website & documentation
+- They help users
+- They can be trusted to decide when code is ready for release, or when to ask someone else to make the judgment
+- They can be trusted to decide when to merge code (if a code contributor) or when to ask someone else to make the judgment
+
+[To learn more see the ASF documentation.](https://httpd.apache.org/docs/)
```

Comment:
Oh I see this came from https://beam.apache.org/contribute/become-a-committer/#an-apache-beam-committer

I'm not sure the name "Beam community practices" is appropriate here, part of it is about community practices, but the entire page is about the qualities of a committer.  Maybe it should be called "Qualities of a Beam Committer" or something. I'll reach out to @griscz about this.

Reply:
Okay, thanks

Reply:
Hi folks, we discussed and the best thing to do is to change the page's title to: 

"Attributes of a Beam community member" 

Instead of Beam community practises

Reply:
done